src/wallet/wallet.h

```diff
@@ -579,11 +579,10 @@ class CWallet : public CCryptoKeyStore, public CValidationInterface
         SetNull();
     }
 
-    CWallet(const std::string& strWalletFileIn)
+    CWallet(const std::string& strWalletFileIn) : strWalletFile(strWalletFileIn)
```

Comment:
Also, are you sure you want that on a new line? Take a look at wallet.h as a whole.

@tgflynn